semantic_router/index/pinecone.py

```diff
@@ -16,15 +17,15 @@ def clean_route_name(route_name: str) -> str:
 class PineconeRecord(BaseModel):
     id: str = ""
     values: List[float]
-    route: str
+    route_name: str
     utterance: str
 
     def __init__(self, **data):
         super().__init__(**data)
         # generate ID based on route name and utterances to prevent duplicates
-        clean_route = clean_route_name(self.route)
+        clean_route_name = clean_route_name(self.route_name)
```

Comment:
there is a function called `clean_route_name` on line `13`, so you can't do this